Replace PIXIT with PICS in log statements

PeterC1965 opened the pull request:
Replace PIXIT with PICS in the log messages.
Fixes #35356

PeterC1965 committed:
Replace PIXIT with PICS in log statements

jamesharrow commented on src/python_testing/TC_EWATERHTR_2_1.py:
```diff
@@ -99,22 +99,22 @@ async def test_TC_EWATERHTR_2_1(self):
             value = await self.read_whm_attribute_expect_success(attribute="TankVolume")
         else:
-            logging.info("Skipping step 4 as PIXIT.EWATERHTR.EM not supported")
+            logging.info("Skipping step 4 as PICS.EWATERHTR.EM not supported")
```
```suggestion
            logging.info("Skipping step 5 as PICS.EWATERHTR.EM not supported")
```

jamesharrow commented on src/python_testing/TC_EWATERHTR_2_1.py:
```diff
@@ -99,22 +99,22 @@ async def test_TC_EWATERHTR_2_1(self):
             asserts.assert_greater_equal(value, 0, f"Unexpected EstimatedHeatRequired value - expected {value} >= 0")
         else:
-            logging.info("Skipping step 5 as PIXIT.EWATERHTR.EM not supported")
+            logging.info("Skipping step 5 as PICS.EWATERHTR.EM not supported")
```
```suggestion
            logging.info("Skipping step 6 as PICS.EWATERHTR.EM not supported")
```

jamesharrow commented on src/python_testing/TC_EWATERHTR_2_1.py:
```diff
@@ -99,22 +99,22 @@ async def test_TC_EWATERHTR_2_1(self):
             asserts.assert_less_equal(value, 100, f"Unexpected TankPercentage value - expected {value} <= 100")
         else:
-            logging.info("Skipping step 6 as PIXIT.EWATERHTR.TP not supported")
+            logging.info("Skipping step 6 as PICS.EWATERHTR.TP not supported")
```
```suggestion
            logging.info("Skipping step 7 as PICS.EWATERHTR.TP not supported")
```

jamesharrow committed:
Apply suggestions from code review

Fixed step numbers being out and made PICS F00(EnergyManagement) / F01(TankPercentage) to be more in line with the test plan.
